pages/index/_learn-more.html

```diff
@@ -38,3 +38,14 @@ <h2>Get expert support and training.</h2>
     <p><a href="/training/" class="btn btn-primary" role="button" ga-on="click" ga-event-category="home-learn-more" ga-event-action="preview-training">Preview Our Training</a></p>
   </div>
 </div>
+
+<div class="row row-cis-compliance">
```

Comment:
Do we have space to add the CIS logo next to the AWS, GCP, and HashiCorp logo? Would make our entire offering looking more secure & production-grade!

Reply:
Good idea - added the logo alongside the others. Will post a screenshot below.